deploy_api.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
```

Comment:
Could add `set -eu` here, as a good practice in bash scripts: `-e` to exit the script right after any failed command, `-u` to exit the script if there are any unset variables such as empty env vars. No variables are used right here, but this is helpful whenever we use a scripts that depends on such. [set man page](https://linuxcommand.org/lc3_man_pages/seth.html)